corehq/apps/userreports/util.py

```diff
@@ -172,3 +173,16 @@ def get_backend_id(config, can_handle_laboratory=False):
     if settings.OVERRIDE_UCR_BACKEND:
         return settings.OVERRIDE_UCR_BACKEND
     return config.backend_id
+
+
+@contextmanager
+def temp_setting_value(name, value):
```

Comment:
Thanks @snopoke. I'll drop this, and deal with the specific use case where it's needed (which is to switch off `CELERY_ALWAYS_EAGER` for a single Celery task, so that a temporary data source is not deleted immediately).